# Custom ros2 message defintion for key-value pairs
int32 key
IntTuple value
